### ISSUE TITLE
Installation documentation

### ISSUE THREAD

**Issue by bakman2** [appmodule/node-red-contrib-openzwave-am#3]
Although I thought I was reasonably technically inclined, these installation instructions are not clear to me tbh

>This package has one sole dependency: node-openzwave-shared. This is a fork of node-openzwave that links to OpenZWave as a shared library, 

Ok - "sole dependency" - node-openzwave-shared, next it states: 

>"you need to have the OpenZWave library installed in your system beforehand"

So these are 2 dependencies, no ?

Next it talks about
> Persisting ZWave configuration in zwcfg_xxx.xml files

What xml files, where do I find these ? Do i need to create them ? What contents ?

Next it states:

```
ozwDriver = new OpenZWave({
        Logging: (logging != "off"),
        ConsoleOutput: (logging != "off"),
        QueueLogLevel: ((logging == "full") ? 8 : 6),
        UserPath: RED.settings.userDir,
        DriverMaxAttempts: cfg.driverattempts
      });

```
Where should one put this ? In settings.js ? In a function node ? 

Long story short: how do I install this node in node-red ? Can this be documented/rewritten in simple steps ? I would love to write it too, but as I don't understand the procedure I can't help at this moment.

**Comment by milosjovac** [comment]
Step 1: Install `libopenzwave1.5`

```bash
apt-get update && apt-get install -y libopenzwave1.5 libopenzwave1.5-dev
```

Step 2: Install `node-red-contrib-openzwave-am`
```bash
cd $HOME/.node-red
npm install node-red-contrib-openzwave-am
```
Or install it using the Node-Red Editor Palette:

![image](https://user-images.githubusercontent.com/2808506/67873443-c21b8c80-fb33-11e9-95bd-53abaa463ff1.png)


PRs are very welcome.
